server/forge/gitea/gitea.go

```diff
@@ -95,12 +88,16 @@ func (c *Gitea) URL() string {
 }
 
 func (c *Gitea) oauth2Config(ctx context.Context) (*oauth2.Config, context.Context) {
+	publicFacingOAuthURL := server.Config.Server.ForgeOAuthHost
+	if publicFacingOAuthURL == "" {
+		publicFacingOAuthURL = c.url
+	}
```

Comment:
Should we add this for all selfhostable forges like gitlab?

Reply:
Yes, I think that would make sense.

Reply:
added it